ts/torch_handler/densenet_handler.py

```diff
@@ -0,0 +1,147 @@
+"""
```

Comment:
@dhanainme is this planned to be aded as default handler? or having it in the example would suffice.